docs/config.md

```diff
@@ -182,6 +182,15 @@ a `$`):
 
 `less -FRX` is the default pager in the absence of any other setting.
 
+### Using more complex pagers
+
```

Comment:
nit: "Filtering pager contents" or "Processing contents to be paged" might be better.

Reply:
Updated!